### DISCUSSION TITLE
Dev Tools load state from JSON not working

### DISCUSSION THREAD

**Discussion by sizemoreap** [vuejs/pinia#1854] [Help and Questions]
I have a store that is populated after a very lengthy process and it's a nightmare to debug. I recently discovered the dev tools have a means of saving the state as a JSON file. SWEET! The issue is that when I load the JSON, the dev tools show me all my variables set but when I run the code that is supposed to process some data, the state variables are empty as though the state somehow wasn't applied; this at the same time the vue tools Pinia plugin is showing me all my data just fine.

Anyone else running into this? Is it not just a simple matter of clicking the load button, selecting the JSON file, and having the state be restored?

**Comment by dignityinside** [comment]
I may have a similar error, check it out: https://github.com/vuejs/pinia/discussions/1860
